docs/compute/data-transfer.md

```diff
@@ -6,7 +6,7 @@ application called _Globus_ or through command-line tools such as
 secure copy (_scp_), secure ftp (_sftp_) and _rsync_. 
```

Comment:
On this page in the `Interactive file transfer with sftp` section there is a table. Could you please highlight the `Command` and `Example` column elements using code highlighting syntax i.e. back ticks?